[AppSec] Remove instances of Build Integrity from Classic (#227)
hlxsites/prisma-cloud-docs: docs/en/classic/appsec-admin-guide/risk-prevention/code/enforcement.adoc

** *Code Reviews*: The *Code Reviews* view provides a real-time monitoring results, offering contextual information about code security across repositories. Utilize graphical representation to comprehend the overall distribution of Failed and Passed scans. You can directly inspect the scan on the version control system (VCS). Further, you can delve into the specifics of the latest scan results and access supplementary details on *Application Security > Projects*.

By combining these two views, you can interpret the scan results, ensuring that you have a comprehensive approach for a quick identification of potential vulnerabilities and facilitate remediation and improvement.

> //Currently the ability to review violation fix suggestions and view the Pull Request (PR) scans that failed is supported only for Github repositories. From the Prisma Cloud console you can directly access your repositories in Github and remediate solutions through a Pull Request (PR).

Author: @@tsmithv11 - Does this statement still hold true?

[.task]

[#review-scan-result-and-suggestions-for-vcs]

> === Review scan results and suggestions for VCS (Version Control System)

Author: @@tsmithv11 - Can I please request a review for this section? This is for the Classic UI and had some inconsistencies.

> To thoroughly examine the scan results of your repositories following the execution of an Enforcement configuration in the Prisma Cloud console, see the steps here for seamless and insightful review.

Reviewer: ```suggestion
To examine the results of your repository scans with the Enforcement configuration in the Prisma Cloud console, see the steps here for seamless and insightful review.
```

[.procedure]

. Accessing Development Pipelines
.. Select *Application Security > Development Pipelines*.
+
image::enfor-14.png[]

. Projects View
+

> By default, you will see the *Projects* view, offering a comprehensive overview of your repositories. This view includes crucial information, such as *Failed PR*, *Pending PR*, and the *Latest PR*. This snapshot is enriched with data on *Weekly commits* and the number of *Git users* interacting with the repository.

Reviewer: ```suggestion
By default, you will see the *Projects* view, offering a helpful overview of your repositories and the pull request activity. This view includes crucial information, such as *Failed PR*, *Pending PR*, and the *Latest PR*. This snapshot is enriched with data on *Weekly commits* and the number of *Git users* interacting with the repository.
```

. Taking Action on Failed or Pending PRs
.. Select the *Actions* menu and choose either:
+
* *Open Failed PR scans on VCS*: This option allows you to directly inspect the failed pull or merge request in the Version Control System (VCS).
* *Review Pending fix PRs in VCS*: Here, you can scrutinize pending fixes for pull or merge requests in the VCS. Additionally, you have the flexibility to accept or reject suggested changes on GitHub.
+
NOTE: Ensure that you have the necessary permissions to make changes in the repository via the VCS.

. Supplementary Details for Repositories
+
..Select *Open the latest scan item* to access more detailed information about the repository directing you to *Application Security > Projects*.

. Code Reviews View

> .. Select *Code Reviews* to gain real-time insights into the security status of repositories post-Enforcement configuration. Here, you have two key *Actions*:

Reviewer: ```suggestion
.. Select *Code Reviews* to gain real-time insights into the security status of repository scans with Enforcement configurations applied. Here, you have two key *Actions*:
```
